\doxysection{scipy.\+io.\+\_\+fortran.\+Fortran\+EOFError Class Reference}
\hypertarget{classscipy_1_1io_1_1__fortran_1_1FortranEOFError}{}\label{classscipy_1_1io_1_1__fortran_1_1FortranEOFError}\index{scipy.io.\_fortran.FortranEOFError@{scipy.io.\_fortran.FortranEOFError}}


Inheritance diagram for scipy.\+io.\+\_\+fortran.\+Fortran\+EOFError\+:
% FIG 0


Collaboration diagram for scipy.\+io.\+\_\+fortran.\+Fortran\+EOFError\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Indicates that the file ended properly.

This error descends from TypeError because the code used to raise
TypeError (and this was the only way to know that the file had
ended) so users might have ``except TypeError:``.
\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{__fortran_8py_source_l00013}{13}} of file \mbox{\hyperlink{__fortran_8py_source}{\+\_\+fortran.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/scipy/io/\+\_\+fortran.\+py\end{DoxyCompactItemize}
